homework/index.js

```diff
@@ -21,27 +21,117 @@
     parent.appendChild(elem);
     Object.keys(options).forEach(key => {
       const value = options[key];
-      if (key === 'text') {
-        elem.innerText = value;
+      if (key === 'html') {
+        elem.innerHTML = value;
       } else {
         elem.setAttribute(key, value);
       }
     });
     return elem;
   }
 
+  function showRepos(leftDiv, repository) {
+    leftDiv.innerHTML = '';
+    const table = createAndAppend('table', leftDiv);
+    const tableBody = createAndAppend('tbody', table);
+    const repositoryTitle = createAndAppend('tr', tableBody);
+    createAndAppend('td', repositoryTitle, { html: 'Repository:', class: 'label' });
+    createAndAppend('a', repositoryTitle, {
+      target: '_blank',
+      href: repository.html_url,
+      html: repository.name,
+    });
+    if (repository.description !== null) {
+      const description = createAndAppend('tr', tableBody);
+      createAndAppend('td', description, {
+        html: 'Description:',
+        class: 'label',
+      });
+      createAndAppend('td', description, { html: repository.description });
+    }
+    const forks = createAndAppend('tr', tableBody);
+    createAndAppend('td', forks, { html: 'Forks:', class: 'label' });
+    createAndAppend('td', forks, { html: repository.forks });
+    const update = createAndAppend('tr', tableBody);
+    createAndAppend('td', update, { html: 'Updated:', class: 'label' });
+    createAndAppend('td', update, { html: repository.updated_at });
+  }
+
+  function showContributors(rightDiv, repository) {
+    rightDiv.innerHTML = '';
+    createAndAppend('p', rightDiv, {
+      html: 'Contributions',
+      class: 'contributor-head',
+    });
+    fetchJSON(repository.contributors_url, (err, contributors) => {
+      if (err) {
+        createAndAppend('div', rightDiv, {
+          html: err.message,
+          class: 'alert-error',
+        });
+      }
+
+      contributors.map(contributor => {
+        const contributorUL = createAndAppend('ul', rightDiv, {
+          class: 'contributor-ul',
+        });
+        const contributorLi = createAndAppend('li', contributorUL, {
+          class: 'contributor-li',
+        });
+
+        const contributorLink = createAndAppend('a', contributorLi, {
+          target: '_blank',
+          href: contributor.html_url,
+        });
+        createAndAppend('img', contributorLink, {
+          class: 'contributor-image',
+          src: contributor.avatar_url,
+        });
+        const contributorDetails = createAndAppend('div', contributorLink, {
+          class: 'contributor-details',
+        });
+
+        createAndAppend('div', contributorDetails, {
+          html: contributor.login,
+          class: 'contributor-names',
+        });
+        createAndAppend('div', contributorDetails, {
+          html: contributor.contributions,
+          class: 'contributors',
+        });
+      });
+    });
```

Comment:
You are generating one `ul` per contributor. Instead, there should be one `ul` for the whole contributor list and an `li` per contributor:

```html
<ul>
  <li>details of first contributor</li>
  <li>details of second contributor</li>
</ul>
```

This is what you generate:

```html
<div class="right_div">
  <p class="contributor-head">Contributions</p>
  <ul class="contributor-ul">
    <li class="contributor-li">
      <a target="_blank" href="https://github.com/gijscor" >
        <img
          class="contributor-image"
          src="https://avatars0.githubusercontent.com/u/15912395?v=4"
        />
        <div class="contributor-details">
          <div class="contributor-names">gijscor</div>
          <div class="contributors">47</div>
        </div>
      </a>
    </li>
  </ul>
  <ul class="contributor-ul">
    <li class="contributor-li">
      <a target="_blank" href="https://github.com/wouterkleijn" >
        <img
          class="contributor-image"
          src="https://avatars1.githubusercontent.com/u/37488847?v=4"
        />
        <div class="contributor-details">
          <div class="contributor-names">wouterkleijn</div>
          <div class="contributors">8</div>
        </div>
      </a>
    </li>
  </ul>
</div>
```